client/my-sites/sidebar/sidebar.jsx

```diff
@@ -366,13 +413,23 @@ export class MySitesSidebar extends Component {
 		);
 	}
 
-	trackUpgradeClick = () => {
+	recordMenuItemClick = menuItemName => {
+		this.props.recordTracksEvent( 'calypso_mysites_sidebar_' + menuItemName + '_clicked' );
```

Comment:
Do we want to add a check here to convert dashes to underscores similar to what you did with [posts/pages](https://github.com/Automattic/wp-calypso/pull/22540/files#diff-43f6fde4305b0c3e2291f0aa4002e6adR132)?

Reply:
Good point. I left it out for these because we know the labels in advance and none contain dashes — but they might in the future. Updated.